### PR TITLE
chore: Add public build

### PR DESCRIPTION
**Pull request by MichaelKowal**

### CODE REVIEW

**Review by gpickell — APPROVED** [review]
Seems OK.
You may need to change the package name.
You may want to set the matrix to only ubuntu-latest, 20

File: .github/workflows/ci-cd.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+env:
+    CI: true
+
+jobs:
+    test:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v3
+
+            - name: Use Node.js 18
```
Comment (rcooney) [inline .github/workflows/ci-cd.yml]: Please update to `20.x` and `actions/setup-node@v4`

File: .github/workflows/ci-cd.yml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            - master
+
+env:
+    CI: true
+
+jobs:
+    test:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v3
```
Comment (rcooney) [inline .github/workflows/ci-cd.yml:18]: Please update to `actions/checkou@v4`

Comment (rcooney) [reply .github/workflows/ci-cd.yml:18]: And the same in the `release` job.

File: .github/workflows/ci-cd.yml (hunk 1 above)
Comment (rcooney) [reply .github/workflows/ci-cd.yml]: And the same in the `release` job.

File: package.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@vertigis/rollup-plugin-license",
```
Comment (rcooney) [inline package.json]: The package name needs to be `@vertigis`

**Comment by rcooney** [comment]
FYI. I've disabled the `CI` build that is failing. I can turn it back on if needed, but it looks like your plan is to only use the new build. Let me know.

File: .github/workflows/ci-cd.yml [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+            - run: npm run build
+
+            - run: npm run lint
+
+    release:
+        runs-on: ubuntu-latest
+        needs:
+            - test
+        if: github.ref == 'refs/heads/main' || github.ref == 'refs/heads/beta'
```
Comment (rcooney) [inline .github/workflows/ci-cd.yml]: This will need to be `master`. You can probably drop the `beta` condition.

File: .releaserc.json [hunk 5]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (rcooney) [inline .releaserc.json:1]: I believe we will need to move this file into a `.config` folder. There is a breaking change in `semantic-release` `v23` https://github.com/semantic-release/semantic-release/releases/tag/v23.0.0.

File: package.json [hunk 6]
```diff
@@ -12,21 +12,25 @@
...
+    "release": "gulp release:minor",
+    "release:patch": "gulp release:patch",
+    "release:minor": "gulp release:minor",
+    "release:major": "gulp release:major"
   },
   "keywords": [
     "rollup",
     "rollup-plugin"
   ],
   "author": "Mickael Jeanroy <mickael.jeanroy@gmail.com>",
```
Comment (rcooney) [inline package.json]: I think we should be putting our name on these. Go with `VertiGIS` as the author and put `vertigis` back in the URLs. 

We need to credit the author, which we do in the readme, but we are publishing this fork of the package.

**Comment by rcooney** [comment]
Please add the following to the `LICENSE` file on a new line above the original author's copyright statement.

`Copyright (c) 2024 VertiGIS North America Ltd.`

There isn't an official way to do this, but we need to preserve theirs and add ours.

**Review by rcooney — CHANGES_REQUESTED** [review]

File: .releaserc.json (hunk 5 above)
Comment (MichaelKowal) [reply .releaserc.json:1]: It looks like that is only the case for `.releaserc.config.js` files. `.json` should still work ok.

File: package.json (hunk 6 above)
Comment (MichaelKowal) [reply package.json]: Done

File: .github/workflows/ci-cd.yml (hunk 4 above)
Comment (MichaelKowal) [reply .github/workflows/ci-cd.yml]: Done

File: package.json (hunk 3 above)
Comment (MichaelKowal) [reply package.json]: Done

File: .github/workflows/ci-cd.yml (hunk 2 above)
Comment (MichaelKowal) [reply .github/workflows/ci-cd.yml:18]: Done

File: .github/workflows/ci-cd.yml (hunk 1 above)
Comment (MichaelKowal) [reply .github/workflows/ci-cd.yml]: Done

**Review by rcooney — APPROVED** [review]

**Pull request merged**